### ISSUE TITLE
Toggle Component: defaultToggled not being set

### ISSUE THREAD

**Issue by sogoiii** [erikras/redux-form-material-ui#102]
Expect: That when I set defaultToggled, the component should be in the on position per the [properties](http://www.material-ui.com/#/components/toggle)

```javascript
import { Field } from 'redux-form'
import { Toggle } from 'redux-form-material-ui'

export const Upload = (props) => (
<Field
  name='componentName'
  label='componentLabel'
  defaultToggled
  component={Toggle}
  />
)
```

Current: When defaultToggled set, the component is in the off position.

Version:
redux-form-material-ui --> 4.1.2
material-ui --> 0.17.0

**Comment by jmenglis** [comment]
I'm not sure that the fix is delivering the proper expected result. 

According to the [material-ui](http://www.material-ui.com/#/components/toggle) it indicates that either defaultToggled **or** toggled should be used based on if the component is [controlled](https://facebook.github.io/react/docs/forms.html#controlled-components) or [uncontrolled](https://facebook.github.io/react/docs/uncontrolled-components.html).

I am working on the putting together a merge request to adjust this.  Probably should be passing an error/warning as well if they use both in a component.

**Comment by erikras** [comment]
Fix published in [`v4.1.3`](https://github.com/erikras/redux-form-material-ui/releases/tag/v4.1.3).
